corehq/apps/translations/integrations/transifex/client.py

```diff
@@ -222,40 +216,3 @@ def get_project_langs(self):
         """
         language_stats_list = self.api.ResourceLanguageStats.filter(project=self.project)
         return [self._language_id_to_lang_code(stats.language.id) for stats in language_stats_list]
-
-    def move_resources(self, hq_lang_code, target_project, version=None, use_version_postfix=True):
```

Comment:
if memory serves and seeing the [commit message](https://github.com/dimagi/commcare-hq/commit/4001c33d4e3e4bbd283563ab646319b8cd620ba8), this function was added to be used manually, which was used quite extensively back then.
Seems fine to remove it or retain it with a comment about how it can be used.

Reply:
I think if it's intended to be run on a production server without UI access, this would be better placed in a management command. My preference is just to remove it right now, since it doesn't really do anything we can't also do manually.